docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,294 @@
+
+## tabs
+
+```yaml
+tabs:
+  - key: people
+    type: table
+```
+
+Tabs appear in the top right of the exercise and serve as a way to display
+datasets or files. The appropriate asset must be defined in the [course
+manifest](../repo-structure.md). A tab field has two subfields: `key`, which is
+the alias for the asset as defined in the manifest, and `type`, which specifies
+how to render the asset. Currently only `file` and `table` are supported.
+
+# Summary of available blocks
+
+:ok: Allowed
+
+:heavy_check_mark: Required
+
+:x: Not Allowed
+
+| Block             | Reorder            | Tap                | Select Code        | Select Output      | Select Table       | Multiple Choice    |
+|-------------------|--------------------|--------------------|--------------------|--------------------|--------------------|--------------------|
+| key               | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| context           | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               |
+| question          | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               |
+| image             | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               |
+| code              | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :ok:               | :ok:               | :ok:               |
+| output            | :ok:               | :ok:               | :ok:               | :heavy_check_mark: | :ok:               | :ok:               |
+| table             | :ok:               | :ok:               | :ok:               | :ok:               | :heavy_check_mark: | :ok:               |
+| distractor_blanks | :x:                | :heavy_check_mark: | :x:                | :x:                | :x:                | :x:                |
+| distractor_code   | :x:                | :x:                | :heavy_check_mark: | :x:                | :x:                | :x:                |
+| distractor_output | :x:                | :x:                | :x:                | :heavy_check_mark: | :x:                | :x:                |
+| distractor_table  | :x:                | :x:                | :x:                | :x:                | :heavy_check_mark: | :x:                |
+| distractor_text   | :x:                | :x:                | :x:                | :x:                | :x:                | :heavy_check_mark: |
+| feedback          | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               |
+| feedback_wrong    | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               | :ok:               |
+
+# YAML tips and tricks
```

Comment:
Add advice or links to resources on debugging bad YAML. For example, https://yaml-online-parser.appspot.com is a life-saver.

Reply:
Nice